src/window/index.ts

```diff
@@ -134,6 +131,12 @@ export async function openFirstRun() {
       window.show();
     }
   });
+
+  window.webContents.on('preferred-size-changed', (_event, preferredSize) => {
+    window.setSize(324, preferredSize.height);
+    window.setMinimumSize(324, preferredSize.height);
+  });
```

Comment:
Should this happen in `layouts/dialog.vue` instead? (Though I guess it wouldn't have a fixed width there — do we really need one?)

Reply:
Having some functions that create and show the window located under `src/window/index.ts` and another function that resizes in `src/layouts/dialog.vue` is conceptually very distant.. I also expect layouts and pages to be concerned with the actual layout and not the sizing of the window, IMO that should be the responsibility of `src/window/index.ts`. I think these are the reasons why I kind of like having all our methods where they are in this PR.

I'm not 100% satisfied with the implementation we see here, and I'm open to changing it, but I guess I would need some convincing 😆

Reply:
Regarding the width, the value was originally `400` and we decreased it to `324` so that the window would better fit the content as it is rendered on page. 

`preferred-size-changed` isn't providing us a width and I can see why that might be. I think it might be better if we can dynamically set width based on our content, if that is what you are thinking. We do need some value, unless we want some electron default for width.

Reply:
It will be trivial to replace the fixed width if we're willing to replace `dialog` with `div` in `src/layouts/dialog.vue`. I made some changes to this PR to show what that would look like.